README.md

```diff
@@ -17,82 +17,125 @@
+The Publisher service will have the ability to register consumer endpoints to publish events.
 
-HTTP producer example  
-```json
+The transport URL is defined in the format of 
+```yaml
+- "--transport-host=$(TRANSPORT_PROTOCAL)://$(TRANSPORT_SERVICE).$(TRANSPORT_NAMESPACE).svc.cluster.local:$(TRANSPORT_PORT)"
+```
+
+HTTP producer example
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
             - "--transport-host=ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
-            - "--api-port=8089"
+            - "--api-port=9085"
 
+```
+The event producer uses a `pubsubstore` to store Subscriber information, including clientID, consumer service endpoint URI, resource ID etc. These are stored as one json file per registered subscriber. The `pubsubstore` needs to be mounted to a persistent volume in oder to survive a pod reboot.
+
+Example for configuring persistent storage
+
+```yaml
+     spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      serviceAccountName: hw-event-proxy-sa
+      containers:
+        - name: cloud-event-sidecar
+          volumeMounts:
+            - name: pubsubstore
+              mountPath: /store
+      volumes:
+        - name: pubsubstore
+          emptyDir: {}
+          persistentVolumeClaim:
+            claimName: cloud-event-proxy-store
+```
 
+Example PersistentVolumeClaim
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cloud-event-proxy-store
+spec:
+  storageClassName: local-storage
+  resources:
+    requests:
+      storage: 10Mi
+  accessModes:
+  - ReadWriteOnce
 ```
-##### Consumer
+
+#### Consumer
 Consumer application will also set its own `transport-host`, which enabled cloud event proxy to run a service to listen to
 incoming events posted by the publisher.
 Consumer will also use `http-event-publishers` variable to request for registering  publisher endpoints for consuming events.
+
 HTTP consumer example
-```json
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
             - "--transport-host=consumer-events-subscription-service.cloud-events.svc.cluster.local:9043"
             - "--http-event-publishers=ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
-            - "--api-port=8089"
+            - "--api-port=9085"
 ```
 
 ## Creating Publisher
-#### Publisher JSON Example 
+### Publisher JSON Example 
 Create Publisher Resource: JSON request
 ```json
 {
   "Resource": "/east-edge-10/vdu3/o-ran-sync/sync-group/sync-status/sync-state",
-  "UriLocation": "http://localhost:8089/ack/event"
+  "UriLocation": "http://localhost:9085/ack/event"
```

Comment:
UriLocation should be consumer service api port which is request for event subscription , not 9085

Reply:
updated to 9090